.php_cs.cache

```diff
@@ -0,0 +1 @@
ag":true,"phpdoc_no_package":true,"phpdoc_no_useless_inheritdoc":true,"phpdoc_return_self_reference":true,"phpdoc_scalar":true,"phpdoc_separation":true,"phpdoc_single_line_var_spacing":true,"phpdoc_tag_type":{"tags":{"inheritDoc":"inline"}},"phpdoc_to_comment":true,"phpdoc_trim":true,"phpdoc_trim_consecutive_blank_line_separation":true,"phpdoc_types":true,"phpdoc_types_order":{"null_adjustment":"always_last","sort_algorithm":"none"},"phpdoc_var_without_name":true,"semicolon_after_instruction":true,"single_class_element_per_statement":true,"single_line_comment_style":{"comment_types":["hash"]},"single_line_throw":true,"single_quote":true,"single_space_after_construct":true,"space_after_semicolon":{"remove_in_empty_for_expressions":true},"standardize_increment":true,"standardize_not_equals":true,"switch_continue_to_break":true,"trailing_comma_in_multiline_array":true,"trim_array_spaces":true,"unary_operator_spaces":true,"visibility_required":true,"whitespace_after_comma_in_array":true,"blank_line_after_opening_tag":true,"compact_nullable_typehint":true,"declare_equal_normalize":true,"lowercase_cast":true,"lowercase_static_reference":true,"new_with_braces":true,"no_blank_lines_after_class_opening":true,"no_leading_import_slash":true,"no_whitespace_in_blank_line":true,"ordered_class_elements":{"order":["use_trait"]},"return_type_declaration":true,"short_scalar_cast":true,"single_blank_line_before_namespace":true,"single_trait_insert_per_statement":true,"ternary_operator_spaces":true,"blank_line_after_namespace":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"encoding":true,"full_opening_tag":true,"error_suppression":{"mute_deprecation_error":false,"noise_remaining_usages":false,"noise_remaining_usages_exclude":[]},"non_printable_character":true},"hashes":{"altapay.php":68974041,"classes\\MerchantAPI.php":2611360412,"controllers\\front\\callbackfail.php":2083725536,"controllers\\front\\callbackform.php":2508225154,"controllers\\front\\callbacknotification.php":1439388557,"controllers\\front\\callbackok.php":1184855679,"controllers\\front\\callbackopen.php":853231097,"controllers\\front\\callbackredirect.php":1435841904,"controllers\\front\\deletecreditcard.php":202155877,"controllers\\front\\orderconfirmation.php":1941333140,"controllers\\front\\payment.php":825166543,"controllers\\front\\savecreditcard.php":4088146273,"controllers\\front\\showsavedcreditcards.php":115413906,"helpers.php":1709833630,"index.php":2126644304,"lib\\altapay\\altapay-php-sdk\\lib\\helpers.php":3526644319,"Models\\Terminal.php":2650438824}}
```

Comment:
Don't commit your local cache files. Probably add this go git ignore instead.

Reply:
I have added this in the .gitignore file but do we still need to delete this file?

Reply:
yes, else ignoring it won't have any effect.